azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java

```diff
@@ -574,6 +573,8 @@ protected void handleCancelFlow(ExecutionReference executionReference,
       final String finalizingReason = "Unable to gracefully kill the flow execution.";
       ExecutionControllerUtils.finalizeFlow(this.executorLoader, this.alerterHolder,
           executableFlow, finalizingReason, e, finalizingStatus);
+      this.fireEventListeners(Event.create(executableFlow,
```

Comment:
same as above, comment is needed to describe the code change.

Reply:
Done. Thx!